db/seeds.rb

```diff
@@ -26,178 +26,178 @@
+                  price: 16000,
+                  image: open(path + "ex_washington_dc.jpg"))
 
 item13.categories << categories[4]
 item13.save
 
-item14 = Item.new(title: "Cupcakes",
-                  description: "Chocolate-Bacon Cupcakes with Dulce De Lech Frosting",
-                  price: 1100,
-                  image: open(path + "cupcakes.jpg"))
+item14 = Item.new(title: "Capital Hill House",
+                  description: "We like our politicians like we like our house, clean and liberal",
+                  price: 11000,
+                  image: open(path + "ex_washington_dc2.jpg"))
 
 item14.categories << categories[4]
 item14.save
 
-item15 = Item.new(title: "Apple Bacon Galette",
-                  description: "Perfet end to a perfect meal",
-                  price: 1400,
-                  image: open(path + "galette.jpg"))
+item15 = Item.new(title: "Filibuster Flat",
+                  description: "Perfet if you talk a lot",
+                  price: 14000,
+                  image: open(path + "ex_washington_dc.jpg"))
 
 item15.categories << categories[4]
 item15.save
 
-item16 = Item.new(title: "Bacon Skewers",
-                  description: "Maple Chocolate Bacon Skewers",
-                  price: 900,
-                  image: open(path + "skewers.jpg"))
+item16 = Item.new(title: "Lincoln Bedroom",
+                  description: "In the White Horse!",
+                  price: 9000,
+                  image: open(path + "ex_washington_dc.jpg"))
 
 item16.categories << categories[4]
 item16.save
 
-item17 = Item.new(title: "Chocolate Brownies",
-                  description: "Smothered in vanilla icing and bacon!",
-                  price: 1000,
-                  image: open(path + "brownies.jpg"))
+item17 = Item.new(title: "Cherry Blossom B&B",
+                  description: "Stay in on our luxurious lupaddlewheeler river boats on Tidal Basin",
+                  price: 10000,
+                  image: open(path + "ex_washington_dc.jpg"))
 
 item17.categories << categories[4]
 item17.save
 
-item18 = Item.new(title: "Ice Cream",
-                  description: "Bacon infused ice cream",
-                  price: 1400,
-                  image: open(path + "ice_cream2.jpg"))
+item18 = Item.new(title: "Private Ford Theatre Balcony Suite",
+                  description: "Privacy guarenteed, or your money back",
+                  price: 14000,
+                  image: open(path + "ex_washington_dc2.jpg"))
 
 item18.categories << categories[4]
 item18.save
 
-item19 = Item.new(title: "Chipotle Bacon Spicy Pecan Caramel Apples",
-                  description: "Spicy sweet savory crunchy tart crisp",
-                  price: 1100,
-                  image: open(path + "apples.jpg"))
+item19 = Item.new(title: "House of Cards",
+                  description: "Power is a lot like real estate. It’s all about location, location, location. The closer you are to the source, the higher your property value.",
```

Comment:
Im in the same boat, it makes sense to me to just make them pending until we make more changes and we are in the position to correct them. :sparkles: